src/components/structures/ThreadView.tsx

```diff
@@ -188,13 +189,31 @@ export default class ThreadView extends React.Component<IProps, IState> {
         }
     };
 
+    private updateThreadRelation = (): void => {
+        this.setState({
+            lastReply: this.state.thread?.lastReply((ev: MatrixEvent) => {
+                return ev.isRelation(THREAD_RELATION_TYPE.name) && !ev.status;
+            }),
+        });
+    };
+
     private updateThread = (thread?: Thread) => {
         if (thread && this.state.thread !== thread) {
+            if (this.state.thread) {
+                this.state.thread.off(ThreadEvent.NewReply, this.updateThreadRelation);
+                this.props.room.off(RoomEvent.LocalEchoUpdated, this.updateThreadRelation);
+            }
```

Comment:
We should also unsubscribe from the current thread if the new thread is `null`, right?
```suggestion
    private updateThread = (thread?: Thread) => {
        if (this.state.thread === thread) return;

        if (this.state.thread) {
            this.state.thread.off(ThreadEvent.NewReply, this.updateThreadRelation);
            this.props.room.off(RoomEvent.LocalEchoUpdated, this.updateThreadRelation);
        }
        if (thread) {
```